Snake/BusinessLayer/maze.cs

```diff
@@ -1,137 +1,199 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+
 
 namespace BusinessLayer
 {
-     public class maze
+    public class Maze
     {
-        private const int BORDER = 1;
-        private const int NO_BORDER = 0;
+        private const int border = 1;
+        private const int noBorder = 0;
+        private enum direction { horizon, vertical };
         private int width, height;
-        private int[,] Maze;
-        Random random_number;
+        private int[,] maze;
```

Comment:
consider if int is the correct data type for the maze. what if it was nullable Elements instead?